src/utilities/legacy-namespace.ts

```diff
@@ -0,0 +1,31 @@
+export function getProviderInfo(data) {
+  const summary_fields = data.summary_fields;
+  let provider = null;
+
+  if (summary_fields) {
+    if ('provider_namespace' in summary_fields) {
+      // role summary
+      provider = data.summary_fields.provider_namespace;
+    } else if (
+      'provider_namespaces' in summary_fields &&
+      summary_fields.provider_namespaces.length > 0
+    ) {
+      // legacy namespace summary
+      provider = data.summary_fields.provider_namespaces[0];
+    }
+  }
+
+  if (!provider) {
+    return {
+      id: null,
+      name: null,
+      url: null,
+    };
+  }
+
+  return {
+    id: provider.id || null,
+    name: provider.name || null,
+    url: provider.name ? `/namespaces/${provider.name}` : null,
```

Comment:
urls for Link need to be produced by `formatPath`, otherwise they won't work in all the environments..

```suggestion
    url: provider.name ? formatPath(Paths.legacyNamespace, { namespaceid: provider.name })` : null,
```

(`import { Paths, formatPath } from 'src/paths';`)